external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

```diff
@@ -131,11 +130,7 @@ class KafkaTestUtils(
   }
 
   private def setUpMiniKdc(): Unit = {
-    val kdcDir = Utils.createTempDir()
```

Comment:
Can we simply use `eventually`? e.g.:

```scala
eventually(timeout(10.seconds), interval(1.seconds)) {
  ...
}
```

Reply:
Do I still need to address this comment？

Reply:
Yeah, can we just use `eventually` instead of having another class? This is in the guide, see also https://github.com/databricks/scala-style-guide#misc_well_tested_method

Reply:
I agree to use `eventually` in general but the actual code copy-pastes ~20 lines. I think a helper represents value. Without helper several places must be fixed if any modification needed which not all the time discovered.

Reply:
Let's make a helper case when we need to duplicate this one more next time. Two times should be fine with or without it :-).

Reply:
OK